### ISSUE TITLE
May need to add a check for btrfs volume mbsize

### ISSUE THREAD

**Issue by FzCola** [OSInside/kiwi#2210]
Given filesystem="btrfs" with preferlvm="true" and "oem-resize" is true by default. 
The creation of the subvolume /var is reporting the following error during the build process:
DEBUG: 08:43:49 | EXEC: [mkfs.btrfs /dev/vgSystem/LVVar]
DEBUG: 08:43:49 | EXEC: Failed with stderr: ERROR: '/dev/vgSystem/LVVar' is too small to make a usable filesystem
ERROR: minimum size for each btrfs device is 114294784
, stdout: btrfs-progs v5.16.2

The calculated mbsize for /var is 66M, which is small than the minimum size required by btrfs. 
Maybe need to add a check to ensure the mbsize returned from get_volume_mbsize in base.py is large enough for "btrfs" filesystem.

**Comment by FzCola** [comment]
I temporarily add 
`if filesystem_name =='btrfs' and mbsize < 109:
    mbsize = 109`
as a workaround in my local usr/lib/python3.6/site-packages/kiwi/volume_manager/base.py and It works.

**Comment by davidcassany** [comment]
Interesting I believe this is a duplicate of https://github.com/OSInside/kiwi/issues/1835

yes with  `oem-resize` btrfs and lvm volumes this issue may raise if there is no data in the volume. Would it be a reasonable workaround adding some extra file to reach the minimum size btrfs requires?

On the other hand, just out of curiosity, is there any reason to use `btrfs` together with LVM volumes? The common approach is just using btrfs subvolumes, I know it is not the same but for many use cases they just work the same.

**Comment by FzCola** [comment]
> Interesting I believe this is a duplicate of #1835
> 
> yes with `oem-resize` btrfs and lvm volumes this issue may raise if there is no data in the volume. Would it be a reasonable workaround adding some extra file to reach the minimum size btrfs requires?
> 
> On the other hand, just out of curiosity, is there any reason to use `btrfs` together with LVM volumes? The common approach is just using btrfs subvolumes, I know it is not the same but for many use cases they just work the same.

Agree. This should be a dup of #1835. It there any updates to fix this issue? I see that issue was opened one year ago.

In my case, this config is required to do so.

**Comment by davidcassany** [comment]
> It there any updates to fix this issue? I see it was opened one year ago.

I don't think so, unfortunately. But at a time it was an issue I reported after debugging a very weird (I'd even say wrong) xml description someone was struggling with. The issue on user side was solved by using different XML setup (switch lvm to subvolumes or different fs) or by adding some extra file to reach the minimum, I don't remember all details. All in all, the issue did not get much attraction as there wasn't any user asking for it (until now) and it looked like a tiny a corner case.

Probably now it is the time to address it. I'll try to find some time these days to move it on again.
